shannon/testnet/genesis.json

```diff
@@ -0,0 +1,742 @@
+{
+  "app_name": "poktrolld",
+  "app_version": "0.0.11-rc",
+  "genesis_time": "2024-11-19T22:52:46.166451Z",
+  "chain_id": "poktroll",
+  "initial_height": 1,
+  "app_hash": null,
+  "app_state": {
+    "application": {
+      "applicationList": [],
+      "application_list": [],
```

Comment:
This is interesting. We only have `applicationList` in the original config.yml. Don't know where `application_list` is coming from here.

Reply:
Ditto for all other camel cased configs

Reply:
I don't feel comfortable changing this right now.

- Not all modules have repeating fields (in different cases) and I don't know why
- The code might expect a particular field - and I need to understand which one before removing another
- Missing field might cause an issue
- This can be removed with an upgrade in the future if necessary